Compiler/README.txt

```diff
@@ -177,14 +174,11 @@ RESOURCES THAT MADE THIS PROJECT POSSIBLE (Descriptions as listed online):
 
 KNOWN BUGS:
 -------------------------------------------------------------------------------
-We don't have any known bugs but we did fall behind in some development 
-practices this time around. Namely
+We don't have any known bugs at this time. However:
 - We still need to add some more unit test. Our test suite isn't fully
 	exhaustive right now, but we're hoping it will be.
-- The FAKEIT framework is not working on the student.cs server, but does work 
-	locally on our machines, and does pass the TravisCI testing. We believe there
-	is some conflict with the version of g++ compiler on the server and the FakeIt
-	version. (note this works on other g++ compilers, just not on the server)
+- The FAKEIT framework is not working on the student.cs server due to the GCC version
+	on student.cs.uni being 4.7, and FAKEIT needing GCC 4.8
```

Comment:
@hilliardj did you independently confirm that this was the issue? the 4.8 thing?